snuba/utils/schemas.py

```diff
@@ -466,11 +466,13 @@ def __init__(
         self,
         func: str,
         arg_types: Sequence[ColumnType[TModifiers]],
+        simple: bool = False,  # SimpleAggregateFunction is faster for things like sum
```

Comment:
The [list of SimpleAggregateFunction](https://clickhouse.com/docs/en/sql-reference/data-types/simpleaggregatefunction)(s) is quite small, why not either:

*  just detect those functions here and set `simple` automatically
* Create a SimpleAggregateFunction class so that it maps to the clickhouse version

Right now this requires that the user of the class know what `SimpleAggregateFunction` is AND it is hiding it behind an ambiguous flag called `simple`

Reply:
I can set it by default in another PR, was scared of side effects